tests/test_georaster.py

```diff
@@ -907,10 +909,17 @@ def test_reproject(self, example: str) -> None:
         assert np.count_nonzero(r_gaps_reproj.data.mask) == tot_masked_true
 
         # If a nodata is set, make sure it is preserved
-        r_ndv = r.copy()
-        r_ndv.set_ndv(255)
-        r3 = r_ndv.reproject(r2)
-        assert r_ndv.nodata == r3.nodata
+        r_nodata = r.copy()
+
+        # Exception with the Landsat example
+        if np.count_nonzero(r_nodata.data.data == 255) > 0:
```

Comment:
Why is this addition needed?
I'm curious, is the 255 value important here or not? Now I realized that 0 is a more appropriate nodata value, when one needs to be set, as it is not used.

Reply:
Here I was just making the existing test work now that there is a `UserWarning` raised when the nodata value exists, I didn't pick `255`.
I'll change to `0` to avoid the warning, I agree it's better.